### DISCUSSION TITLE
A way to hide all recipes on the homepage dashboard

### DISCUSSION THREAD

**Discussion by ken8521** [mealie-recipes/mealie#1629] [Feature Request]
First, just found this the other day and am truly loving this.  I've manually imported about 10-12 recipes and having them in one place like this makes it super easy.

There's two particular things I think could be better (or maybe I'm missing how to do this the way I want).

1.  The shopping list feature is fantastic.  I can easily create one manually or create one off of a recipe if I'm picking up for just one recipe.  Just really well done.  However, I don't see anyway to export it to a file format I could read on my phone when not at home on my network.  I have reverse proxy'd my mealie instance through my domain, and the webUI is easy enough on a mobile browser, it makes going through my shopping list at the store very simple.  This is no big deal to me, but some sort of export feature would be nice.  Unfortunately it leads to issue 2, since I'm kind of a privacy minded person

2.  When I go to my domain (mealie.mydomain.tld).. I have to login to see my shopping list, create/edit recipes, etc.  Unfortunately from best I can tell... Even not logged in, any recipe, category, etc. is accessible without an account on my domains dashboard.  Someone who just stumbled across my domain could just read and download them without permission (but not edit them).  Now it's not that I particularly care about some random downloading my breakfast casserole recipe, but I'd still rather these things be private unless I choose to make them public.

1 Isn't a huge issue.. it was really more just a thought.  

2  I consider an issue (maybe most here don't)

Again, fantastic work, I'm really glad I ran across this container the other day.  Is there a plan for an Android app in the future to access our servers?

**Comment by tehniemer** [comment]
Which version are you running? I think the v1.0 betas force a login before any recipes are shown.

**Reply by michael-genson** [reply]
This is correct, unless there's a bug we don't know about. V1.0.x requires sign-in to see anything (unless you're on the most recent nightly where you can opt-in to public recipe links)

**Reply by ken8521** [reply]
I'm on .5.6 acccording to the UI
 

I just deployed it Sunday.. is there a label  I need to add to make it pull the most recent version? (also using watchtower which has ran several times since I installed it)

**Reply by michael-genson** [reply]
Check out the nightly docs for 1.0.xb, I strongly recommend this version over 0.5.6:

https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/

**Reply by ken8521** [reply]
Woo.. big changes.  Obviouslky I installed a old version (that's what was on the website).  This seems to have addressed my primary concern!  Thanks!

**Comment by michael-genson** [comment]
See @tehniemer's  comment above about recipe privacy. Regarding the shopping list, I've been working on expanding the capabilities of our integrations so we can integrate with other lists/apps. An export or print feature sounds like a good idea. Is there a particular app you use for shopping, or are you just thinking about a spreadsheet/pdf?

**Reply by ken8521** [reply]
I think a PDF would be fine....

If you want the ability to "check off" items... the Notes app on Android has a nice checklist feature (not sure if that could be integrated or not)

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing this one. The shopping list features are much improved since then.
